tests/forms_tests/tests/test_forms.py

```diff
@@ -2298,6 +2298,35 @@ class UserRegistration(Form):
 <span class="helptext">Wählen Sie mit Bedacht.</span></td></tr>"""
         )
 
+        p = UserRegistration()
+        self.assertHTMLEqual(
+            p.as_ul(),
+            """<li><label for="id_username">Username:</label>
+<input type="text" name="username" maxlength="10" required aria-describedby="id_username_helptext" id="id_username">
+<span class="helptext" id="id_username_helptext">e.g., user@example.com</span></li>
+<li><label for="id_password">Password:</label>
+<input type="password" name="password" required aria-describedby="id_password_helptext" id="id_password">
+<span class="helptext" id="id_password_helptext">Wählen Sie mit Bedacht.</span></li>"""
+        )
+        self.assertHTMLEqual(
+            p.as_p(),
+            """<p><label for="id_username">Username:</label>
+<input type="text" name="username" maxlength="10" required aria-describedby="id_username_helptext" id="id_username">
+<span class="helptext" id="id_username_helptext">e.g., user@example.com</span></p>
+<p><label for="id_password">Password:</label>
+<input type="password" name="password" required aria-describedby="id_password_helptext" id="id_password">
+<span class="helptext" id="id_password_helptext">Wählen Sie mit Bedacht.</span></p>"""
+        )
+        self.assertHTMLEqual(
+            p.as_table(),
+            """<tr><th><label for="id_username">Username:</label></th><td>
+<input type="text" name="username" maxlength="10" required aria-describedby="id_username_helptext" id="id_username">
+<br><span class="helptext" id="id_username_helptext">e.g., user@example.com</span></td></tr>
+<tr><th><label for="id_password">Password:</label></th><td>
+<input type="password" name="password" required aria-describedby="id_password_helptext" id="id_password"><br>
+<span class="helptext" id="id_password_helptext">Wählen Sie mit Bedacht.</span></td></tr>"""
+        )
+
```

Comment:
Can we move it to a separate test method? We already have so many HTML assertions here.

Reply:
Will do. 

Also do you have a prefrence on triple quotes vs single quotes on each line. (I think you tweeked one of my recent patches to use single quotes on each line).

Reply:
We use `assertHTMLEqual()` so newlines don't matter, I would use single quotes:
```python
        self.assertHTMLEqual(
            p.as_table(),
            '<tr><th><label for="id_username">Username:</label></th><td>'
            '<input type="text" name="username" maxlength="10" required '
            'aria-describedby="id_username_helptext" id="id_username">'
            ...
       )
```

Reply:
This assertions are now redundant with `test_help_text_aria_describedby()` and can be removed.